pages/common/xmllint.md

```diff
@@ -1,24 +1,24 @@
 # xmllint
 
-> XML parser and linter.
+> XML parser and linter that supports XPath, a syntax for navigating XML trees.
 
-- Return all nodes named "foo":
+- Return all nodes (tags) named "foo":
 
 `xmllint --xpath "//{{foo}}" {{source_file.xml}}`
 
-- Return as string the contents of first node named "foo":
+- Return, as a string, the contents of the first node named "foo":
```

Comment:
Maybe this is better - "Return the contents of the first node named "foo" as a string"